src/server/test-compiler.es6

```diff
@@ -23,6 +23,10 @@ function writeContent(path, content) {
 function compile(body) {
   return `
     var assert = require("assert");
+    require = function() { throw new Error("require not available") };
+    module.require = require;
```

Comment:
There is module.require method which is somewhat a backdoor:

https://nodejs.org/api/modules.html#modules_module_require_id

I am not completly sure this could be a problem, but just in case...